framework/source/class/qx/ui/basic/Label.js

```diff
@@ -535,7 +535,12 @@ qx.Class.define("qx.ui.basic.Label",
     _applyValue : function(value, old)
     {
       // Sync with content element
-      this.getContentElement().setValue(value);
+      if (value.translate) {
```

Comment:
the result of this is a LocalizedString ... which will react to changing locales ... often :+1: